### PR TITLE
testing/py-biopython: new aport

### PR DESCRIPTION
**Pull request by Downchuck**
depends on numpy

### CODE REVIEW

**Comment by Downchuck** [comment]
based on the APKBUILD of py-pandas as reference for a numpy dependent package

**Comment by maxice8** [comment]
squash all into 1 commit with a proper title for new aports

**Comment by maxice8** [comment]
new packages go in testing/

**Comment by Downchuck** [comment]
@maxice8  thx - done.

**Comment by maxice8** [comment]
Can it be python3 only ? py3-biopython. We are currently working on completely removing python2 from Alpine

**Comment by Downchuck** [comment]
I've tripped over a legacy package here and there that's py2. It's certainly easier on my life to maintain py2 support through end of year ... But that said, I will make an effort to get any items I'm in converted.

Do you want me to just do python 3, and squash the commit again, or can we let this through while in testing?

**Comment by maxice8** [comment]
Just python3 and squash the commits, i won't be pushing anything that adds py2

**Comment by Downchuck** [comment]
@maxice8 may the python3 be ever in your favor

File: testing/py-biopython/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# Maintainer: Charles Pritchard <chuck@jumis.com>
+pkgname=py-biopython
+_pkgname=biopython
+pkgver=1.74
+pkgrel=0
+pkgdesc="Python tools for computational molecular biology."
+url="http://biopython.org/"
+arch="all"
+license="BSD-3-Clause"
+depends="py-numpy"
```
Comment (maxice8) [inline testing/py-biopython/APKBUILD]: py3-numpy

File: testing/py-biopython/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+makedepends="linux-headers py-numpy-dev python3-dev py-setuptools"
```
Comment (maxice8) [inline testing/py-biopython/APKBUILD]: py3-setuptools

Comment (Downchuck) [reply testing/py-biopython/APKBUILD]: you still want this package named py-biopython?

**Comment by maxice8** [comment]
No, py3-biopython

**Comment by Downchuck** [comment]
Maybe got it right this time.

**Comment by maxice8** [comment]
Pushed with the changes:

- Commit title corrected (py-biopython -> py3-biopython)
- added check()
- Added options="!check" because test_NCBI_qblast hangs
- fixed url

**Pull request closed without merging**